tests/e2e/specs/dashboard-samples/RecommendedExtensions.spec.ts

```diff
@@ -81,6 +81,9 @@ for (const sample of samples) {
 		});
 
 		test('Check the project files were imported', async function (): Promise<void> {
+			// add 20 sec timeout for waiting for finishing animation of all IDE parts (Welcome parts. bottom widgets. etc.)
```

Comment:
I would not mention 20 sec timeout in here, because TIMEOUT_CONSTANTS.TS_IDE_LOAD_TIMEOUT could be redefined using environment variable when run the test.